x/rollup/keeper/msg_server.go

```diff
@@ -33,14 +33,6 @@ func (k *Keeper) ApplyL1Txs(goCtx context.Context, msg *types.MsgApplyL1Txs) (*t
 
 	ctx.Logger().Info("Save L1 block info", "l1blockInfo", string(lo.Must(json.Marshal(l1blockInfo))))
 
-	// save L1 block History to AppState
-	if err = k.setL1BlockHistory(&ctx, l1blockInfo); err != nil {
-		ctx.Logger().Error("Failed to save L1 block history info to AppState", "err", err)
-		return nil, types.WrapError(types.ErrL1BlockInfo, "save error: %v", err)
-	}
```

Comment:
Now that we're no longer storing the L1 block info by hash we'll want to remove any unit test assertions that check that it's properly stored.

https://github.com/polymerdao/monomer/blob/main/x/rollup/keeper/msg_server_test.go#L132